### ISSUE TITLE
Tables cannot be explored with keyboard navigation

### ISSUE THREAD

**Issue by parseccentric** [gelic-idealab/komodo-portal#37]
**Expected behavior**
Table rows and table rows' buttons are navigable by pressing tab on a keyboard.

**Actual behavior**
Inconsistent:
* In "Admin > Users" and "Admin > Courses", action buttons and table pagination controls are focusable, but not checkboxes
* In "Dashboard > Assets" and the "Assets" page itself, only the search box and table pagination controls are focusable. Importantly, the rows themselves are clickable, but not focusable by the keyboard
* In "[Course] > [Lab] > Edit Lab Details > Assets" and "[Course] > Schedule New Lab", only the search box and table pagination controls are focusable 

**How to reproduce**
On any page with a table, try to navigate the table with a keyboard only. 

Thank you @ho-yi-shiuan for bringing this up!

**Comment by parseccentric** [comment]
Limitations of the built-in component provided by Vue are revealed on the demo page: https://v2.vuetifyjs.com/en/components/data-tables/ 

You can see how checkboxes are not focusable at all. You can also see how action buttons are focusable by default.

As for clicking on a row, I recommendation is that we deprecate the ability to click on the row itself in favor of having an new action button that does the same thing. We have had several users attempt to click the checkbox or the action buttons instead of the row itself, so this could eliminate some confusion.

As for selecting a row (like in Edit Lab Details), I do not know the best way forward.

**Comment by wallacerob** [comment]
Yeah, I don't even remember why we have checkboxes for Labs in the first place! I like the idea of having action buttons for the various modes, "Edit", "View", "Delete", etc.

**Comment by wallacerob** [comment]
Also, yeah the assets table has always seemed a little out of place on the dashboard.

**Comment by parseccentric** [comment]
Haha, yeah. We probably should have removed it when we took out the metrics summary. One day we'll have a cool AI dashboard B)

**Comment by parseccentric** [comment]
Oh also, @ho-yi-shiuan, a couple things:

1. Just a pre-emptive reminder that opening an issue is more to keep track of the conversation, and not necessarily to say "work on this right now" -- just so you don't feel like assignments are coming in left and right :)
2. When you said that putting tabindex on a table would select the whole table, what would happen if you pressed enter on that table? Probably nothing, but part of me hopes that it magically would focus a row, somehow.

**Comment by ho-yi-shiuan** [comment]
For 2, actually the whole table isn't able to be selected, although it has a tabindex, I can't select it with tab.
